indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/IndyUser.kt

```diff
@@ -214,6 +215,7 @@ open class IndyUser : IndyIssuer, IndyProver, IndyTrustee {
             if (getRootCause(e) !is DuplicateMasterSecretNameException) throw e
 
             logger.debug("MasterSecret already exists, who cares, continuing")
+            logger.info { "" }
```

Comment:
Do we need it?

Reply:
It is removed and rearranged in the coming PR.

Reply:
sounds good